src/main/java/uk/gov/hmcts/darts/usermanagement/controller/UserController.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.darts.usermanagement.controller;
+
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.darts.authorisation.annotation.Authorisation;
+import uk.gov.hmcts.darts.usermanagement.http.api.UserApi;
+import uk.gov.hmcts.darts.usermanagement.model.UserWithIdAndLastLogin;
+
+import java.util.List;
+
+import static uk.gov.hmcts.darts.authorisation.constants.AuthorisationConstants.SECURITY_SCHEMES_BEARER_AUTH;
+import static uk.gov.hmcts.darts.authorisation.enums.ContextIdEnum.ANY_ENTITY_ID;
+import static uk.gov.hmcts.darts.common.enums.SecurityRoleEnum.ADMIN;
+
+@RestController
+public class UserController implements UserApi {
+
+    @Override
+    @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
+    @Authorisation(contextId = ANY_ENTITY_ID, globalAccessSecurityRoles = ADMIN)
```

Comment:
Would this be more suited to a new context courthouse id?

Reply:
I'm not sure it needs a new contextId?  I only used `ANY_ENTITY_ID` here as I needed to pick one in the current Authorisation annotation. Unless we change the contextId to be an array with a default like the others so that I can just have `@Authorisation(globalAccessSecurityRoles = ADMIN)`
but then I'd have to mess about with the validation we have in our AuthorisationAspect or introduce a new annotation for `@GlobalAuthorisation`.

I didn't think it was worth me changing the User parameter endpoints for this example, although I do think the parameters should be renamed to be more consistent with other endpoints now that they are integer e.g. `courthouse_id`